### ISSUE TITLE
UI and writing updates to make the WCAG 2 page more readable and usable

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-website#799]
Hi there, I really want to send my clients to an official resource that defines WCAG as they get started. However, I often find people would rather use unofficial sites like [wcag.com](https://wcag.com/) than this page. Though there are likely SEO reasons for their popularity as well, I do find the unofficial sites provide the direct, plain language and content segments (e.g., WCAG by role) that people new to the subject find helpful.

I wonder, then, if a few UI and writing changes would help this WCAG 2 page better meet user needs. Below are some thoughts/suggestions:

1. **Define WCAG right up-front:** The opening Summary & Page Contents section takes up a good deal of space leaving a user to scroll for the most important piece - a definition for WCAG. Often I even hesitate recommending this page because I know it will initially look like the page doesn't have content, just links. I wonder if, instead, the first sentence of the Summary could be a plain definition of WCAG, then an explanation of the remainder of the page, then the links. 
2. **Collapse the list of on-page links ("Page content" section):** Related to the opening content, because it is so long I wonder if an expand/collapse can help. Obviously the data about page views should be consulted, but I suspect this page is meant to be most helpful for first-time visitors; thus, I think the focus should be on putting content up-front and easily digestible, not necessarily quick links to content that the user isn't yet familiar with. If the page is then revisited, I think options for those users such as quick links should be a secondary consideration because they will be more motivated to use those tools.
3. **Add or revise how WCAG is defined:** The current content shown under the "Introduction" heading starts with how WCAG is created, not what it is - and the remaining content never quite plainly defines it as well. I think this is an important first step in talking about it and causes people to then create their own definitions (including myself). For instance, in my trainings I often say, "WCAG is a list of criteria that help define and measure accessibility for websites and mobile applications." Essentially I try to explain that WCAG provides a starting point for trying to define what it means for technology to be accessible so that we can get the work of accessibility done. Something about how this is presented on this page doesn't quite hit home the same way and doesn't create quotable content I find I can use in my presentations. 

Happy to chat more about my comments and help with updating it, including collecting feedback from others which may help inform the updates. Thanks for considering!
